src/calculatingBullsAndCows.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
+
+function calculatingBullsAndCows(randomNumber, userNumber) {
+  const userNumberArray = userNumber.toString().split('').map(num => +num);
+  const randomNumberArray = randomNumber.toString().split('').map(num => +num);
+
+  let count = 0;
+  const bulls = [];
+  const cows = [];
+
+  for (let i = 0; i < userNumberArray.length; i++) {
+    if (userNumberArray[i] === randomNumberArray[i]) {
+      count++;
+      bulls.push(userNumberArray[i]);
+      continue;
+    }
+
+    const index = randomNumberArray
+      .findIndex(num => userNumberArray[i] === num);
+
+    if (index >= 0 && index !== i) {
```

Comment:
Check if `index >= 0` is redundant here. `i` will not be equal to `index` in case `index` is `-1` anyway